fedimint-server/src/config/mod.rs

```diff
@@ -366,6 +376,11 @@ impl ServerConfig {
         let private = self.private.clone();
         let id = identity.to_usize();
 
+        let my_public_key = private.broadcast_secret_key.public_key(&Secp256k1::new());
```

Comment:
Creating a new secp context every time is a bit of an anti-pattern. Not sure if using the [global context](https://docs.rs/secp256k1/0.27.0/secp256k1/static.SECP256K1.html) would be worse though since we are dealing with secret data here.